### DISCUSSION TITLE
1.x版本，x-component用的treeselect，treedata是通过接口请求的，但是组件内容一直是空的

### DISCUSSION THREAD

**Discussion by tunshiyu** [alibaba/formily#1673] [Q&A 1.x]
在schemaform层加个1s的setTimeout强制更新下，是有数据的。所以怀疑是执行顺序问题。所以想用useAsyncDataSource处理一下试试，但是怎么设置进去呢，看文档的样子是给select用的，treeselect如何设置到treeData中
![image](https://user-images.githubusercontent.com/39406845/123728683-42c0fc00-d8c6-11eb-84b9-f62eabb31d25.png)

**Comment by janryWang** [comment]
给个复现demo
